src/providers/companionServer.js

```diff
@@ -185,9 +186,9 @@ const serverFunction = (req, res) => {
             let data = {
                 player: infoPlayerProvider.getPlayerInfo(),
                 track: infoPlayerProvider.getTrackInfo(),
-                /*queue: infoPlayerProvider.getQueueInfo(),
+                queue: infoPlayerProvider.getQueueInfo(),
                 playlist: infoPlayerProvider.getPlaylistInfo(),
-                lyrics: infoPlayerProvider.getLyricsInfo(),*/
+                lyrics: infoPlayerProvider.getLyricsInfo(),
```

Comment:
Have you reviewed that these work considered they were commented and no changes to these appear to be made in infoPlayerProvider?